.github/workflows/release.yml

```
@@ -66,13 +66,69 @@ jobs:
         run: npm ci --legacy-peer-deps
 
-      - name: Verify workspace
+      # This used to be `npm run verify` (typecheck && test && build:inspector),
+      # a serial ~5-8 min re-run of work `test.yml` and `lint.yml` already did
+      # on this exact commit when it landed on main — and it built the SDK three
+      # times over. Gating on those workflows' results instead takes seconds and
+      # covers strictly more: `lint.yml` runs typecheck, typecheck:client,
+      # build:inspector and a production boot; `test.yml` runs the suites.
+      # BOTH are required — neither alone covers what `verify` did.
+      - name: Require green CI for candidate SHA
         if: ${{ !fromJSON(inputs.skip_verify) }}
```

actionlint (1.7.12)

[error] 1st argument of function call is not assignable. "bool" cannot be assigned to "string". called function type is "fromJSON(string) -> any"

(expression)